tests/cli/superset/sync/dbt/lib_test.py

```diff
@@ -386,6 +386,12 @@ def test_filter_models() -> None:
         "unique_id": "model.three",
         "depends_on": ["source.zero"],
         "children": ["model.two"],
+        "config": {
```

Comment:
Looks like the tests are failing because you need to add `config` to the other models, even if it's an empty dict.